kbase-extension/static/kbase/js/util/timeFormat.js

```diff
@@ -1,14 +1,25 @@
-/*global define*/
-/*jslint white: true*/
 /**
  * A few utility functions. These are kinda unrelated, but used in several places.
  *
  * @author Bill Riehl wjriehl@lbl.gov
  */
-define([], function() {
+define([], () => {
```

Comment:
ran eslint + prettier. The usual. No actual code changes, but looked at the file to ensure that the tests were checking the right thing.